model/otel/metrics.yaml

```diff
@@ -0,0 +1,105 @@
+groups:
+  - id: metric.otel.sdk.span.created_count
```

Comment:
We can calculate the number of created spans from the ended and live spans, right?  For processed/exported metrics we take that approach--can we do the same here and remove the created spans metric here?